tests/manifests/__init__.py

```diff
@@ -283,3 +300,8 @@ def _get_deployables_details_from_base_components_names(base_components_names: l
     if any([deployable_details.has_ingress for deployable_details in deployables_details])
     and values_file not in secret_values_files_to_test
 ]
+_extra_services_values_files_to_test = ["matrix-rtc-exposed-services-values.yaml", "matrix-rtc-host-mode-values.yaml"]
+
+services_values_files_to_test = [
+    values_file for details in all_components_details for values_file in details.secret_values_files
```

Comment:
Is this deliberate? Seeding `**services**_values_files_to_test` with `**secret**_values_files`

Reply:
Thanks for catching this!